src/db/meta.rs

```diff
@@ -38,65 +40,106 @@ pub struct Meta {
 
     pub master_key_change_force: Option<isize>,
 
+    /// memory protection settings
     pub memory_protection: Option<MemoryProtection>,
 
+    /// custom icons
     pub custom_icons: CustomIcons,
 
+    /// whether the recycle bin is enabled
     pub recyclebin_enabled: Option<bool>,
 
     /// A UUID for the recycle bin group
     pub recyclebin_uuid: Option<String>,
 
+    /// last time the recycle bin was changed
     pub recyclebin_changed: Option<NaiveDateTime>,
 
+    /// UUID of the group containing entry templates
     pub entry_templates_group: Option<String>,
 
+    /// last time the group containing entry templates was changed
     pub entry_templates_group_changed: Option<NaiveDateTime>,
 
+    /// last selected group UUID
```

Comment:
@sseemayer to keep the same format as `entry_templates_group`:
```
UUID of the last selected group
```

Reply:
Agreed and changed in a follow-up commit.